test/fuzz/fuzz-udev-rules/60-persistent-storage.rules

```diff
@@ -86,7 +86,7 @@ KERNEL=="vd*[0-9]", ENV{ID_PATH}=="pci-*", SYMLINK+="disk/by-path/virtio-$env{ID
 
 # probe filesystem metadata of optical drives which have a media inserted
 KERNEL=="sr*", ENV{DISK_EJECT_REQUEST}!="?*", ENV{ID_CDROM_MEDIA_TRACK_COUNT_DATA}=="?*", ENV{ID_CDROM_MEDIA_SESSION_LAST_OFFSET}=="?*", \
-  IMPORT{builtin}="blkid --offset=$env{ID_CDROM_MEDIA_SESSION_LAST_OFFSET}"
+  IMPORT{builtin}="blkid --hint session_offset=$env{ID_CDROM_MEDIA_SESSION_LAST_OFFSET}"
```

Comment:
we usually specify the "=" for cmdline options that take params, i.e. please add the extra = here too:

```
blkid --hint=session_offset=…
```

Reply:
Fixed.